juriscraper/lib/string_utils.py

```diff
@@ -182,7 +205,7 @@ def fix_camel_case(s):
         s_out = s
     else:
         s_out = s[0]
-        for i in xrange(1, len(s)):
+        for i in six.moves.range(1, len(s)):
```

Comment:
Should we just make this `range` instead of `xrange`? Remove the need for `six`?